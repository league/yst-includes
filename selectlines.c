#include <stdio.h>

int main()
{
  printf("Hello, world.\n");
  if(rand()%2 == 0) {
    printf("Yes\n");
  } else {
    printf("No\n");
  }
  return 0;
}

void frob(int x)
{
  // BEGIN EXAMPLE
  int y = x + 2;
  printf("%d %d\n", x, y);
  // END
}

float grab()
{
  // BEGIN EXAMPLE
  printf("OK\n");
  return 3.14;
  // END
}
