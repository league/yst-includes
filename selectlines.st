
We can select certain lines only.

~~~~ {.c include="selectlines.c" first="5" last="10"}
~~~~

