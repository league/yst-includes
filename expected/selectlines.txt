We can select certain lines only.

~~~~ {.c}
  printf("Hello, world.\n");
  if(rand()%2 == 0) {
    printf("Yes\n");
  } else {
    printf("No\n");
  }
~~~~
